test/unit/server/services/viperService.spec.js

```diff
@@ -0,0 +1,53 @@
+import createViperService from '../../../../server/services/viperService';
+
+describe('viper service', () => {
+  let fakeDB;
+  let viperService;
+
+  function setup() {
+    fakeDB = { raw: x => x };
+    fakeDB.select = sinon.stub().returns(fakeDB);
+    fakeDB.table = sinon.stub().returns(fakeDB);
+
+    viperService = createViperService(fakeDB);
+  }
+
+  describe('selecting a viper rating', () => {
+    before(() => setup());
+
+    it('returns a viper rating for a known nomis id', () => {
+      fakeDB.where = sinon.stub().resolves([{ nomisId: 'A123', rating: 0.90 }]);
+
+      return viperService.rating('A123')
+        .then((result) => {
+          expect(fakeDB.select.callCount).to.eql(1);
```

Comment:
You only call the viperService once in your code.. can we remove the call counts in your test?